core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/RenameConnector.java

```diff
@@ -113,6 +113,10 @@ public Response rename(@QueryParam("oldPath") String oldPath,
       // Clarify new name & check to add extension
 
       String newName  = Text.escapeIllegalJcrChars(newTitle);
+
+      //clean node name
+      newName = Utils.cleanName(newName);
```

Comment:
Ok this will help renaming contents, whet should we do with other documents having quotes in their names ?

Reply:
For uploading docs this is already handled : 
https://github.com/exoplatform/ecms/blob/7e780b752e8798c2a747e2836f9631bf438780d1/core/connector/src/main/java/org/exoplatform/wcm/connector/FileUploadHandler.java#L417

for the affected renamed doc before this fix, we need only to rename them again, I already merged a PR to allow displaying then context menu for docs with double quote char